src/makeInfinityAdder.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  return function adder(a) {
+    if (a >= 0) {
```

Comment:
I'm not sure it's a good idea to compare a and b with 0. This function will not work with negative numbers. You can just check if they are not undefined

Reply:
Good point. Thanks!